package.json

```diff
@@ -66,7 +66,8 @@
     "koa2-cors": "^2.0.6",
     "lodash": "^4.17.15",
     "log4js": "^6.7.1",
-    "mongoose": "6.*",
+    "mongodb": "~5.9.1",
+    "mongoose": "^7.6.7",
```

Comment:
❓ since orka is in 4.0 beta, do we want to make it only mongo7 compatible? @nikostoulas WDYT?

Reply:
Sure! Let's use 7.* to not restrict apps from getting latest updates

Reply:
👍 I'll use the `7.*` notation for clarity